packages/server/src/models/EventModel.ts

```diff
@@ -16,11 +15,11 @@ export default class EventModel extends BaseModel<Event> {
 		return UuidType.Native;
 	}
 
-	public async create(type: EventType, name = '') {
+	public async create(type: EventType, name = '', created_time = Date.now()) {
```

Comment:
This is not necessary and should be avoided. Instead you can use this in the test to simulate time:

jest
  .useFakeTimers()
  .setSystemTime(new Date('2020-01-01'));

See here for more information: https://stackoverflow.com/a/63377110/561309